### ISSUE TITLE
fisherman?

### ISSUE THREAD

**Issue by ghost** [Poorchop/dotfiles#1]
Hi @Poorchop, I'm no longer maintaining omf, and would appreciate a ton if you can upgrade to fisherman:

• fisherman/fisherman#69 (comment)

I'm willing to help you transition anything. Cheers.

Happy fishing!
<github.com/brj>

**Comment by Poorchop** [comment]
Thanks for the info, I'll give it a try. The fish files here are very outdated regardless, but I appreciate your offer to help with transitioning.

**Comment by ghost** [comment]
Absolutely, I'm just a ping away :)

EDIT: There is a [slack](http://join.fisherman.sh) room too.

**Comment by ghost** [comment]
@Poorchop Hey, how's it going? Did you have a chance to try fisherman?

**Comment by Poorchop** [comment]
Yeah it's really great. The recent fish 2.3.0 update has been buggy for me so I'm just waiting for the next release before uploading my changes since it broke my theme.

**Comment by ghost** [comment]
@Poorchop Cool, thanks. fisherman should work fine with 2.3.0 since that's what I am running too. If you have any issues whatsoever, fisherman related or not, maybe I can help, just let me know.

**Comment by Poorchop** [comment]
Thanks. Fisherman works fine, it's just that 2.3.0 seems to be having issues over ssh as well as with messing up theme colors. I'm just waiting on [this issue](https://github.com/fish-shell/fish-shell/issues/2980).
